README.md

```diff
@@ -0,0 +1,94 @@
+# TCTrack
+
+The TCTrack package provides the functionality for running tropical cyclone (TC)
+tracking algorithms and analysing the results. This can be used to compare the output
+of different algorithms for a variety of data sources.
+
+The software is currently in early development and therefore subject to significant
+changes.
+
+For the full documentation, see the website:
+[https://tctrack.readthedocs.io](https://tctrack.readthedocs.io).
+
+
+## Installation
+
+### Dependencies
+The package requires python3 (>=3.9).
+
+### Package Installation
+First clone the repository using `git`:
+```sh
+git clone https://github.com/Cambridge-ICCS/TCTrack
+```
+
+Setup and activate a virtual environment:
+```sh
+python3 -m venv .venv
+source .venv/bin/activate
+```
+When finished using TCTrack this can be turned off with `deactivate`.
+
+Then install the package using `pip`:
+```sh
+pip install TCTrack
```

Comment:
```suggestion
pip install .
```

Reply:
Consider adding a comment for developers about creating an editable install?
Perhaps under development section below.